HHS/Head-Start-TTADP
# [TTAHUB-1557] - Header alignment and responsive behavior

## Description of change

Adjusted css classes to align header content, and add responsive behavior. Also added css changes to support responsive behavior.

## How to test
Look at header on different pages and resize window or use device emulator to see responsive behavior (header is shorter, logo + logo text + avatar is smaller)

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1557


## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
